docs/modules/ROOT/pages/mutations/create.adoc

```diff
@@ -96,3 +96,48 @@ mutation {
 ----
 
 This will create a User with name "John Doe", an introductory Post, both of which will be returned with their autogenerated IDs.
+
+== `connectOrCreate` relations
+If a related node has a `@unique` directive defined, `connectOrCreate` can be used in a
+nested create to perform a `MERGE` operation on the related node, creating a new
+relation and the related node if it doesn't exists.
+
+Consider the following type definition:
+[source, graphql, indent=0]
+----
+type Actor {
+    name: String!
+    movies: [Movie] @relationship(type: "ACTED_IN", direction: OUT)
+}
+
+type Movie {
+    title: String
+    id: String! @unique
```

Comment:
I think documenting this example like this encourages "bad" behaviour, as the `@id` directive also leads to a unique node property constraint being added, with the added bonus of ID values being generated automatically. Perhaps give an example that uses both the `@id` and `@unique` directives, and then point out that you can use either of them to match the node to connect to.

```suggestion
    id: String! @id
```